src/geometry.rs

```diff
@@ -420,7 +420,7 @@ impl fmt::Display for Face {
 
 /// A triangular face. Contains indices to other geometry data, such
 /// as vertices and normals.
-#[derive(Debug, Clone, Copy, PartialEq, Eq, serde::Serialize)]
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash, serde::Serialize)]
```

Comment:
Is this `Hash` needed?

Reply:
Not sure right now, but the whole hassle with the face was for the possibility of deduplication and HashMaps / HashSets. Maybe we don't have any right now, but we certainly will.
Should I remove it now? I'm worried we'll forget it is possible when we need it.

Reply:
Keep it if you're sure we'll use it.